packages/stylelint-config/stylelint.config.js

```diff
@@ -4,12 +4,12 @@ module.exports = {
   plugins: [
     "stylelint-prettier",
     "stylelint-order",
-    "stylelint-color-format",
     "stylelint-declaration-block-no-ignored-properties",
   ],
   rules: {
     "prettier/prettier": true,
-    "color-format/format": { format: "rgb" },
+    "color-named": "never",
+    "color-no-hex": true,
```

Comment:
The latest version of `stylelint-color-format` does not support any version of `stylelint` above `v9.x.x` so we had to get rid of it.

Instead, `stylelint` gives us:
- `color-named` (see documentation for `never` [here](https://stylelint.io/user-guide/rules/color-named/#never))
- `color-no-hex` (see `true` documentation [here](https://stylelint.io/user-guide/rules/color-no-hex/#true))

which will replace what we needed from `stylelint-color-format`